public/index.html

```diff
@@ -3,7 +3,7 @@
   <head>
     <meta charset="utf-8" />
     <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
```

Comment:
Not allowing zooming is bad for users who struggle to read the text and wish to zoom in to see it. This probably makes it feel less app like, but is better overall.

Reply:
This brings back the zooming in mobile safari but I have a better fix for that below.